addons/bestja_project_hierarchy/models.py

```diff
@@ -43,6 +43,7 @@ class ProjectInvitation(models.Model):
         compute_sudo=True,
     )
     invited_projects = fields.One2many('bestja.project', inverse_name='parent_invitation')
+    inviter = fields.Boolean(string="Zapraszający", compute='_compute_inviter')
```

Comment:
The name doesn't reflect the function of the field - based on the name the field should contain id of organization/user who created the invitation. It should at least be called "is_inviter", or in this case even "user_is_inviter" to reflect the fact that the field is dependent on the current user (which is kinda weird ;)).